shell/browser/net/proxying_websocket.cc

```diff
@@ -410,8 +405,7 @@ void ProxyingWebSocket::OnHeadersReceivedCompleteForAuth(
 
   auto continuation = base::BindRepeating(
```

Comment:
Auth requests for websocket proxy is not handled correctly, we don't expose any hooks through the webrequest module but still this is left in IO pending state. Will handle in a separate PR.